modules/odc-grouping-multiple-components.adoc

```diff
@@ -11,20 +11,25 @@ The following procedure adds a MariaDB database service to an existing applicati
 
 .Prerequisites
 
-* Ensure that you have created and deployed a Node.js application on {product-title} using the *Developer* perspective.
+* Ensure that you have created and deployed minimum two or more components on {product-title} using the *Developer* perspective.
 
 .Procedure
 
-. Create and deploy the MariaDB service to your project:
-.. In the *Developer* perspective, navigate to the *+Add* view and select the *From Catalog* option to view all available services in the *Developer Catalog*.
-.. Select *Databases* to list all the database services and then click *MariaDB* option to see the details for the service.
-.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and then click *Create* to create and view the MariaDB service in the *Topology* view.
+. To add a service to the existing application group, press kbd:[Shift]+ drag it to the existing application group
```

Comment:
```suggestion
* To add a service to the existing application group, press kbd:[Shift]+ drag it to the existing application group.
```
The Mod docs template states: ". Use an unnumbered bullet (*) if the procedure includes only one step."

Reply:
Need some clarity on this will discuss in the Gmeet.

Reply:
This suggestion combines the step on line 18 with the result on line 20, below. If you combine them like this, you can delete line 20. See my comment on line 20.

Reply:
@rolfedh Got it, but here I want to explain that once you dragged the component into the application group then it will add the required labels of the application group in the settings of that component. And this adding of labels is now done automatically.

Reply:
@rolfedh To avoid this confusion I'll add the other line as a **NOTE** with a screenshot.